index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" type="text/css" href="./style.css">
+        <title>Intuit accounts - Sign up</title>
+    </head>
+    <body>
+        <div class="header">
+            <span class="header-span" id="sumspace">
+                <p class="header-span"> Already have an Intuit account?</p> 
+                <a href="#">I forgot my User ID or Password</a>
+            </span>
+            <div id="button">
+                <button id="signin-button">
+                    <span class="whitelock"></span>
+                    Sign In
+                </button>
+            </div>
+        </div>
+        <header>
+            <img src="logos/main-logo.png" alt="Intuit">
+        </header>
+        <header class="logos-box">
```

Comment:
You don't need to use 2 headers tags inside the same container. Use just one header tag and use 2 div to separate the main logo from the other logos.